Resources/doc/index.rst

```diff
@@ -50,7 +50,7 @@ Require the bundle with composer:
 
 .. code-block:: bash
 
-    $ composer require friendsofsymfony/user-bundle "~2.0@dev"
+    $ composer require friendsofsymfony/user-bundle "dev-master"
```

Comment:
Event;
use FOS\UserBundle\Model\GroupInterface;
use Symfony\Component\EventDispatcher\Event;
use Symfony\Component\HttpFoundation\Request;
class GroupEvent extends Event
{
    private $group;
    private $request;
    public function __construct(GroupInterface $group, Request $request)
    {
        $this->group = $group;
        $this->request = $request